pandas/io/parsers/readers.py

```diff
@@ -337,6 +337,8 @@ class _read_shared(TypedDict, Generic[HashableT], total=False):
     - "mixed", to infer the format for each element individually. This is risky,
         and you should probably use it along with `dayfirst`.
 
+    You can specify the format for each column by passing a dict
```

Comment:
Could you explain this in the `Example` section below instead?

Reply:
Sure